### PR TITLE
PR #328 ActivateAsync regression fix

### PR DESCRIPTION
**Pull request by kornman00**
Fix LayoutItemStatePersister.LoadState so that it has matching behavior with ShellViewModel.OnViewLoaded. This is due to aa7a8b5557a716f6e3027e46b904c53f80b2e46b (from a PR #318 that I originally made) adding support for calling a Tool's ActivateAsync on the first showing.

This was broken by 60ede6456f2048f77966d899945dc1cc433aa6bf in PR #328.

### CODE REVIEW

**Comment by kornman00** [comment]
The Pack job failed in the CI build because PR #345 isn't merged yet, which bumps the min framework to net462.

**Comment by tgjones** [comment]
Thanks!

**Comment by ReMinoer** [comment]
Hi @kornman00. :)
I [answered](https://github.com/tgjones/gemini/pull/328#issuecomment-1229450603) at your comment on my incriminated pull request #328 to challenge a bit this pull request change.

TLDR of my answer: in my opinion, ActivateAsync should not be called in AddTool. And there is indeed an issue with tool activation and I have an opened pull request #342 to fix it.
